src/components/WorkPoolQueueEditForm.vue

```diff
@@ -107,7 +107,7 @@
 <style>
 .work-pool-queue-edit-form { @apply
   border
-  border-gray-300
+  dark:border-background-600
```

Comment:
if this was wrapped in a card would this just work?

Reply:
Yeah I think there's a small-ish ticket to create a common interface for full-page forms